src/widgets/health-monitoring/HealthMonitoringFileSystem.tsx

```diff
@@ -26,10 +31,10 @@ const OpenMediaVaultFileSystem = ({ fileSystem }: any) => {
             </Group>
             <Group>
               <Text fz="lg" fw={500}>
-                Used: {disk.used} - {disk.percentage}%
+                {t('general.used')}: {disk.used} - {disk.percentage}%
```

Comment:
Use translations with variables. Some languages will have a drastically different syntax.
Ditto on any string that is followed by a variable string.